framework/tlvf/test/main.cpp

```diff
@@ -169,16 +174,19 @@ int main(int argc, char *argv[])
     CmduMessageRx invmsg;
     auto invheader     = invmsg.parse(invalidBuffer, invalidBufferSize, false);
     if (invheader == nullptr) {
-        MAPF_ERR("HEADER PROTECTION SUCCESS");
+        MAPF_INFO("HEADER PROTECTION SUCCESS");
     }
 
     if (!received_message.getNextTlvType(type)) {
-        MAPF_ERR("TYPE PROTECTION SUCCESS");
+        MAPF_INFO("TYPE PROTECTION SUCCESS");
     }
     auto invptr = invmsg.addClass<tlv1905NeighborDevice>();
     if (invptr == nullptr) {
-        MAPF_ERR("PROTECTION SUCCESS");
+        MAPF_INFO("PROTECTION SUCCESS");
+    } else {
+        MAPF_ERR("PROTECTION FAILED");
+        errors++;
     }
 
```

Comment:
Maybe add a print of how many errors was found during the execution

Reply:
Again, nothing blocking. The "errors" variable is actually used as the return code from main to mark test failure - so as long as it is not 0 it will be considered a failure, we don't really care how many errors happened as long as there is more than one - this will fail the test so that when we finally enable CTEST it will serve its purpose.